docroot/modules/custom/foia_users/foia_users.module

```diff
@@ -77,13 +94,12 @@ function foia_users_node_access($node, $op, $account) {
  * Custom submit handler for office value.
  */
 function _form_validation_foiaoffice($form, $form_state) {
```

Comment:
We should change the comments above to reference 'agency' instead of 'office', we should also change the function name.